src/Components/reviewContainer.jsx

```diff
@@ -1,27 +1,42 @@
 import { useState, useEffect } from "react";
 import { listOfReviews } from "../utils/api";
 import { Link } from "react-router-dom";
+import { useLocation} from "react-router-dom";
+
+
+
 
 export default function ReviewContainer() {
   const [list, setList] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
-
+  const {search} = useLocation()
+  const [searched,setSearched]=useState()
+  const params = search.replace("?","")
+  
   useEffect(() => {
-    listOfReviews().then((response) => {
+    listOfReviews(params).then((response) => {
    setList(response)
-      setIsLoading(false);
+   setIsLoading(false);
     });
-    },[]);
-
+  },[searched]);
```

Comment:
Great to see you've added this dependency into the array, if I'm being critical then I'm not exactly sure what it's meant to be related to, I guess it's if the category changes in the URL address bar? It might be worth making the purpose of this variable name clearer.